.ci_support/osx_64_python3.11.____cpython.yaml

```diff
@@ -1,5 +1,7 @@
 MACOSX_DEPLOYMENT_TARGET:
-- '10.9'
+- '10.13'
+MACOSX_SDK_VERSION:
+- '10.13'
```

Comment:
We are setting the deployment target to 11.0 when we link against CasADi for the IDAKLU solver, and for the gfortran dynamic libraries when we link against SuiteSparse – I updated that in https://github.com/pybamm-team/PyBaMM/pull/4128. While I think it is safe to keep 10.13 here because i. I noticed that CasADi's recipe is built against 10.13 (it's silently broken for their PyPI wheels, by the way – see https://github.com/casadi/casadi/issues/3698), and ii. we don't provide the IDAKLU solver through conda-forge yet like we do for the PyPI wheels; we should get this verified with @conda-forge/core, based on the documentation provided about bumping: https://conda-forge.org/docs/maintainer/knowledge_base/#requiring-newer-macos-sdks.

What I have in mind is that if we are requiring a target of 11.0 (across both Intel and Arm) and, therefore, lesser levels of compatibility through PyPI wheels, I'm not sure if it makes sense to provide wider compatibility through the conda-forge channel in comparison.

P.S. For context for other reviewers: the minimum target is 11.0 for Silicon devices anyway, but we set it to 11.0 for Intel ones as well instead of the regular 10.9 for PyPI, and I'm proposing to set it to 11.0 instead of 10.13 here in a similar manner.

Reply:
If something fails to build or run with the default deployment target, then you can increase it. But CI here seems to be green despite targeting 10.13.

Does that mean there is no problem? Or can you add a test for something you expect to fail?

Reply:
Thanks for the suggestion; CasADi being imported and running is a fine test (also that no CI provider provides a macOS version that old, nowadays). The idea was that CasADi's import would fail silently on macOS 10.9–10.16 for its PyPI wheels, which would also break compatibility for PyBaMM. Since CasADi is being built from source (instead of using a `pip`-install command) in its feedstock and builds with the deployment target set to 10.13 pass there, it's indeed safe, and I have no concerns.